core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaLevelEventTranslator.java

```diff
@@ -346,6 +367,35 @@ public void translate(GeyserSession session, ClientboundLevelEventPacket packet)
                 session.sendUpstreamPacket(soundEventPacket);
                 return;
             }
+            // Half of these are listed in MCProtocolLib as Trail instead of Trial for some reason
```

Comment:
![image](https://github.com/GeyserMC/Geyser/assets/105284508/cafdedc4-e69f-4888-9676-052079f727d9)
Yeah, looks like an issue in the cloudburst/protocol - which btw is *not* mcprotocollib, that's the java library. I'll file an issue there

Reply:
my bad, i get confused with the names lol